### DISCUSSION TITLE
Can I set the nickname for all members?

### DISCUSSION THREAD

**Discussion by feri-irawan** [discordjs/discord.js#10038] [Q&A] [answered]
I was trying to put some words in the members' nicknames in my server, but it gives me the error "missing permission."

I tried using the `Administrator` permission for the bot in Discord Developer Portal > Application > OAuth2 > URL Generator and invited my bot, but it still gives me the same error. Then I tried enabling all permissions in the URL generator and enabling the `Administrator` permission in the Server Settings > Roles > MyBotRoles > Permissions, but it also gives me the same error.

My code:
```ts
import { Interaction } from "discord.js";

export async function rename(interaction: Interaction) {
  const members = await interaction.guild?.members.fetch();

  members.each(async (member) => {
    console.log(
      `Manageable: ${member.manageable}, Nickname: ${member.nickname}, Display name: ${member.displayName}`,
    );
    const word = "new word";
    await member.setNickname(`${member.displayName} - ${word}`);
  });

  await interaction.reply("OK");
}
```

**Comment by almostSouji** [comment, marked as answer]
You cannot manage the guild owner or members of the same or a higher role than the bot's highest role, regardless of permissions.

**Reply by feri-irawan** [reply]
I see thank you bro.
